src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -159,7 +159,7 @@ struct EvolutionMetavars {
                           tmpl::list<>>,
       dg::Events::Registrars::ObserveFields<1, Tags::Time, observe_fields,
                                             analytic_solution_fields>,
-      Events::Registrars::ObserveTimeStep<EvolutionMetavars>,
+      Events::Registrars::ObserveTimeStep<system>,
```

Comment:
We've been favoring actions which depend explicitly on the parts of the metavariables that they really need, rather than templating on the whole metavariables -- this was advice I received on a previous PR from @kidder , but my understanding is that like other kinds of code dependencies, it is preferable for the template dependence to be as narrow as it can be so that dependencies are clearer.